tests/Tests/200__monitor_and_manage/200__metrics/201__billing_metrics.robot

```diff
@@ -34,105 +84,191 @@ Begin Billing Metrics Web Test
   RHOSi Setup
 
 End Web Billing Metrics Test
-  CleanUp JupyterHub
-  SeleniumLibrary.Close All Browsers
+    CleanUp JupyterHub
+    SeleniumLibrary.Close All Browsers
+
+Test Setup For Matrics Web Test
+    Set Library Search Order    SeleniumLibrary
+    Open Browser    ${OCP_CONSOLE_URL}    browser=${BROWSER.NAME}    options=${BROWSER.OPTIONS}
+    Login To OCP
+    Wait Until OpenShift Console Is Loaded
+    Click Button    Observe
+    Click Link    Metrics
+    Wait Until Element is Visible    xpath://textarea[@class="pf-c-form-control query-browser__query-input"]
+
+Test Teardown For Matrics Web Test
+    SeleniumLibrary.Close All Browsers
+
+Run Query On Metrics And Return Value
+    [Documentation]    Fires query in metrics through web browser and returns value
+    [Arguments]    ${query}    ${count_of_columns}    # count of columns + 1 like name,values example: ${count_of_columns}=3
+    Input Text    xpath://textarea[@class="pf-c-form-control query-browser__query-input"]    ${query}
+    Click Button    Run queries
+    Wait Until Element is Visible    xpath://table[@class="pf-c-table pf-m-compact"]    timeout=15seconds
+    @{data} =    Get WebElements    //table[@class="pf-c-table pf-m-compact"] //tbody/tr/td[${count_of_columns}]
+    [Return]    ${data[0].text}
+
+Fire Query On Prometheus And Return Value
+    [Documentation]    Fires query in Prometheus through cli and returns value
+    [Arguments]    ${query}
+    ${expression} =    Set Variable    ${query}&step=1    #step = 1 beacuase it will return value of current second
+    ${query_result} =    Prometheus.Run Query    ${RHODS_PROMETHEUS_URL}    ${RHODS_PROMETHEUS_TOKEN}    ${expression}
+    [Return]    ${query_result.json()["data"]["result"][0]["value"][-1]}
 
 Skip Test If Previous CPU Usage Is Not Zero
-  ${version-check} =  Is RHODS Version Greater Or Equal Than  1.5.0
-  IF  ${version-check}==True
-      ${metrics_value} =   Run OpenShift Metrics Query    ${METRIC_RHODS_CPU}
-  ELSE
-      ${metrics_value} =   Run OpenShift Metrics Query    ${METRIC_RHODS_CPU_BEFORE_1.5.0}
-  END
-  ${metrics_query_results_contain_data} =  Run Keyword And Return Status     Metrics.Verify Query Results Contain Data
-  IF  ${metrics_query_results_contain_data}
-    Log To Console    Current CPU usage: ${metrics_value}
-    Skip if  ${metrics_value} > 0  The previos CPU usage is not zero. Current CPU usage: ${metrics_value}. Skiping test
-  END
+    ${version-check} =    Is RHODS Version Greater Or Equal Than    1.5.0
+    IF    ${version-check}==True
+        ${metrics_value} =    Run OpenShift Metrics Query    ${METRIC_RHODS_CPU}
+    ELSE
+        ${metrics_value} =    Run OpenShift Metrics Query    ${METRIC_RHODS_CPU_BEFORE_1.5.0}
+    END
+    ${metrics_query_results_contain_data} =    Run Keyword And Return Status    Metrics.Verify Query Results Contain Data
```

Comment:
I think we can remove this check since v1.5.0 is far in the past now

Reply:
done